src/ADSB/ADSBVehicle.h

```diff
@@ -20,6 +20,31 @@ class ADSBVehicle : public QObject
     Q_OBJECT
 
 public:
+    enum class Emitter : uint8_t {
```

Comment:
Emitter wasn't clear to me what it meant when I saw it elsewhere. Can we call it EmitterType instead?

Reply:
Changed